theme/redbox.js

```diff
@@ -1,4 +1,4 @@
-(function redBoxButton() {
+function redBoxButton() {
```

Comment:
I think this should still be an IIFE, but at the end should have something like
```js
  ...
  window.redboxButtonClicked = redboxButtonClicked;
})();
```

Similarly, `save-playgrounds.js` would have
```js
  ...
  window.resetPlaygroundsClicked = resetPlaygroundsClicked;
})();
```

Then `instructor-menu.js` can have
```js
  redboxButton.addEventListener("click", e => window.redboxButtonClicked(e));
  playgroundStateButton.addEventListener("click", e => window.resetPlaygroundsClicked(e));
```

with all of the implementation of what to do when those buttons are clicked appearing in the appropriate file.